pkg/api/customization/podsecuritypolicytemplate/validator.go

```diff
@@ -0,0 +1,27 @@
+package podsecuritypolicytemplate
+
+import (
+	"errors"
+	"github.com/rancher/norman/httperror"
+	"github.com/rancher/norman/types"
+	"github.com/rancher/norman/types/convert"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+	"k8s.io/kubernetes/pkg/apis/policy"
+	"k8s.io/kubernetes/pkg/apis/policy/validation"
+)
+
+// Validator uses k8s Pod Security Policy validation to prevent creation of pod security policy templates that are invalid
+func Validator(request *types.APIContext, schema *types.Schema, data map[string]interface{}) error {
+	var spec policy.PodSecurityPolicySpec // k8s psp
+	if err := convert.ToObj(data, &spec); err != nil {
+		return httperror.WrapAPIError(err, httperror.InvalidBodyContent, "Pod Security Policy spec conversion error")
+	}
+	var fldPath field.Path
```

Comment:
This is passed in but otherwise unused, that seems wrong, is there information we can get from this field or can it be omitted?

Reply:
Its required for the function call but it is not used.

Reply:
Resolved by putting inline